sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -1623,14 +1782,20 @@ def get_test_params(cls, parameter_set="default"):
         if parameter_set == "results_comparison":
             return {"n_estimators": 3, "max_depth": 2, "n_stump_evaluations": 2}
         else:
-            param1 = {"n_estimators": 2, "max_depth": 1, "n_stump_evaluations": 1}
-            param2 = {
+            params1 = {
+                "n_estimators": 2,
+                "max_depth": 1,
+                "n_stump_evaluations": 1,
+                "backend": "loky",
+            }
+            params2 = {
                 "n_estimators": 4,
                 "max_depth": 2,
                 "n_stump_evaluations": 3,
                 "distance_measure": "dtw",
+                "backend": "dask",
```

Comment:
this causes `dask` to be loaded, even when it is not there, that causes the failures.

So, you may want to not test dask if it is not present, e.g., have a separate parameter set depending on `_check_soft_dependencies` with `dask` (severity = none).